### PR TITLE
Documentation Restructure

### PR DESCRIPTION
**Pull request by donn**
~ Documentation all moved under `docs/source`, arranged hierarchically according to the table of contents
~ `Klayout` changed to `KLayout` in all logging messages
~ Readme rewritten to just be concise, parts of it isolated into standalone documentation
~ RTD builds no longer use conda (saves some time)
~ Fixed all broken links

---

This isn't strictly part of this PR, but I also activated RTD for pull requests. You can press the details button down below to preview everything I did:

image

### CODE REVIEW

File: docs/source/for_developers/docs_contribution.rst [hunk 1]
```diff
@@ -106,8 +105,8 @@ In order to improve the readability of the documentation, please use and capital
     Mac OS X → macOS
     MAGIC → Magic
     Skywater130 → sky130
-    Klayout → KLayout
-    Pip -> pip or PIP 
+    KLayout → KLayout
```
Comment (kareefardi) [inline docs/source/for_developers/docs_contribution.rst:108]: i think this is not needed

Comment (donn) [reply docs/source/for_developers/docs_contribution.rst]: You are right

File: docs/source/usage/chip_integration.md [hunk 2]
```diff
@@ -59,8 +59,8 @@ You need to set ***all environment variables mentioned in [Hardening the Core](#
 The following inputs are provided to produce the final GDSII:
 
 1. Padframe cfg file (provided by the user or generated by padring). [Here][6] is an example. Or a hardened chip_io.gds and chip_io.lef following [this][4].
-2. Hardened lef & GDS-II for the core module, generated [here](#hardening-the-core)
-3. Top level netlist instantiating pads and core module (Could be provided by the user or generated by [topModuleGen][7])
+2. Hardened lef & GDSII for the core module, generated [here](#hardening-the-core)
+3. Top level netlist instantiating pads and core module (Could be provided by the user or generated by [topModuleGen])
```
Comment (kareefardi) [inline docs/source/usage/chip_integration.md:63]: link broken

**Comment by kareefardi** [comment]
`Klayout` changed to `KLayout` in all logging messages.

**Review by kareefardi — APPROVED** [review]

**Comment by armleo** [comment]
"Installation, the long version" leads to 404 @donn

**Pull request merged**